data-service/src/main/java/uk/gov/gchq/palisade/service/data/model/DataRequest.java

```diff
@@ -26,10 +26,10 @@
 import java.util.StringJoiner;
 
 /**
- * The {@link DataRequest} is the input for the data-service where the resource is read.
- * This message is created by the response from the filtered-resource-service to the client.
- * It is then routed via the resource's connectionDetail to the appropriate instance of a data-service.
- * This message is used to retrieve the {@link uk.gov.gchq.palisade.reader.request.DataReaderRequest} from the attribute-masking-service.
+ * The DataRequest represents the client's request for resource after is has been prepared by the Palisade services.
```

Comment:
```suggestion
 * The DataRequest represents the clients request for a resource after is has been prepared by the Palisade services.
```

Reply:
kept the possessive for client and added the a